packages/markers/src/browser/problem/problem-contribution.ts

```diff
@@ -119,6 +155,41 @@ export class ProblemContribution extends AbstractViewContribution<ProblemWidget>
         }
     }
 
+    protected addToClipboard(content: string): void {
+        const handleCopy = (e: ClipboardEvent) => {
+            if (e.clipboardData) {
+                document.removeEventListener('copy', handleCopy);
```

Comment:
please move it to be the first statement, just to make sure that it removes and does not break global copy handlers

Or will such change cause some issues?

Reply:
Hi Anton, by "moving it to be first statement", do you mean moving `document.removeEventListener` to out of the *if* branch, to be the first statement of `handleCopy` (so that it gets removed in all cases)?

Reply:
yes